packages/offchain-resolver-backend/src/http/profile.ts

```diff
@@ -23,17 +24,16 @@ export function profile(web3Provider: ethers.providers.BaseProvider) {
                 const { signedUserProfile, siweMessage, siweSig, hotAddr } =
                     req.body;
 
+                const siwe = new SiweMessage(JSON.parse(siweMessage));
```

Comment:
It would be handy if we can add logs for siweMessage, to enhance debugging

Reply:
also if the app breaks during production it likely occurs here. Maybe we can add some additional monitoring

Reply:
Also, it might sense to wrap the Json.parse in a try-catch because we're not sanitizing the input at all and the server would crash with 500 if siweMessage is not al valid JSON string